src/main/java/com/redhat/cloud/policies/app/health/ProcSelfStatusExporter.java

```diff
@@ -111,51 +128,6 @@ void gather() {
         }
     }
 
-    @Gauge(name = "status.vmHwm", absolute = true, unit = MetricUnits.KILOBYTES, tags = "type=proc")
```

Comment:
Can we preserve the tags with the new gauges?

Reply:
We could. This would change it a bit. registry.gauge(..) would become: Gauge.builder("status_isDegraded", () -> StuffHolder.getInstance().getStatusInfo().size())
.tags(...)
.register(registry)

Right now we don't make use of the tags in our grafana graphs. That's why I did it in the simplest possible way (registry.gauge(...) and removed the tags.

Reply:
Grafana is only a subset of the real metrics usage. For instance, I use plenty of metrics from Prometheus that are not in Grafana when I investigate/monitor performances or a bug on stage/prod.

I didn't add nor use (yet) these tags or the `@SimplyTimed` discussed above so I don't have a strong feeling about keeping or removing them, but they were added for a reason. I'm not sure this migration should alter any existing metric (as long as it can be migrated).

Reply:
I asked Heiko. He added it.

We can either replace it with @Timed annotation or use Antonin Stefanutti's extension, put the @Metrics annotation on the class and the Timed annotation on the respective method.